### ISSUE TITLE
Clonflict with Ackis Recipe List/ElvUI

### ISSUE THREAD

**Issue by vmwinckler** [niketa-wow/farming-bar#13]
**Describe the bug**
Addon don't work in specific combination

**Client**
client 8.3.0

**To Reproduce**
After login:
```
6x ...rfaceFarmingBar\Modules\ObjectiveBuilder.lua:157: attempt to compare string with number
[string "@FarmingBar\Modules\ObjectiveBuilder.lua"]:157: in function <...rfaceFarmingBar\Modules\ObjectiveBuilder.lua:157>
[string "=[C]"]: in function `sort'
[string "@FarmingBar\Modules\ObjectiveBuilder.lua"]:157: in function `CreateObjectiveIconSelector'
[string "@FarmingBar\Modules\ObjectiveBuilder.lua"]:17: in function `CreateObjectiveBuilder'
[string "@FarmingBar\Core\Core.lua"]:12: in function <FarmingBar\Core\Core.lua:10>
[string "=[C]"]: ?
[string "@AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0-12.lua"]:70: in function <...s\AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0-12.lua"]:527: in function `EnableAddon'
[string "@AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0-12.lua"]:620: in function <...s\AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0.lua:605>
[string "=[C]"]: in function `LoadAddOn'
[string "@ElvUI_SLE\modules\professions\professions.lua"]:39: in function <ElvUI_SLE\modules\professions\professions.lua:36>
[string "=[C]"]: ?
...
[string "=[C]"]: in function `Initialize'
[string "@ElvUI\init.lua"]:141: in function <ElvUI\init.lua:140>
[string "=[C]"]: ?
[string "@AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0-12.lua"]:70: in function <...s\AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0.lua:65>
[string "@AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0-12.lua"]:527: in function `EnableAddon'
[string "@AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0-12.lua"]:620: in function <...s\AckisRecipeList\libs\AceAddon-3.0\AceAddon-3.0.lua:605>
[string "=[C]"]: in function `LoadAddOn'
[string "@FrameXML\UIParent.lua"]:457: in function `UIParentLoadAddOn'
[string "@FrameXML\UIParent.lua"]:580: in function `TimeManager_LoadUI'
[string "@FrameXML\UIParent.lua"]:1267: in function <FrameXML\UIParent.lua:1164>
```

When reset addon cache, works for one login... after crash again

**Comment by niketa-wow** [comment]
I've downloaded Ackis Recipe List (and I already use ElvUI), but cannot replicate the error you're receiving.

Can you give me any more information about what's going on?
Can you also tell me all of the addons you currently have installed?

**Comment by niketa-wow** [comment]
Also, are there other errors in addition to the one you posted?

**Comment by vmwinckler** [comment]
nothing... every time, i reset for work and after reload stop work. I try
delete WTF folder and reconfigure all, but problem continue... well.. i
disable farming bar and install older farmIt for test... and surprise...
works fine... i rly dont know...

Em ter., 23 de jun. de 2020 às 14:15, Niketa <notifications@github.com>
escreveu:

**Comment by niketa-wow** [comment]
Since I was unable to replicate this, I've attempted a fix but I have no way of testing it out. I added a tostring call on table values when sorting, where this error was popping up.

Fix included in upcoming v2.1.
